assembl/static2/js/app/components/administration/voteSession/modulesSection.jsx

```diff
@@ -26,17 +47,55 @@ const DumbModulesSection = ({ tokenModules, editLocale, handleCheckBoxChange }:
             <Checkbox
               checked={tokenModuleChecked}
               onChange={() => {
-                handleCheckBoxChange(tokenModuleChecked, tModule[0]);
+                handleTokenCheckBoxChange(tokenModuleChecked, tModule[0], newId);
               }}
             >
               <Helper
                 label={I18n.t('administration.voteWithTokens')}
                 helperUrl="/static2/img/helpers/helper4.png"
                 helperText={I18n.t('administration.tokenVoteCheckbox')}
-                classname="inline"
+                classname="inline checkbox-title"
               />
             </Checkbox>
             {tokenModules.map(id => <TokensForm key={id} id={id} editLocale={editLocale} />)}
+            <Checkbox
+              checked={gaugeModuleChecked}
+              onChange={() => {
+                handleGaugeCheckBoxChange(gaugeModuleChecked, gModule, newId);
+              }}
+            >
+              <Helper
+                label={I18n.t('administration.voteWithGages')}
+                helperUrl="/static2/img/helpers/helper3.png" // TODO: add an actual screenshot
+                helperText={I18n.t('administration.gaugeVoteCheckbox')}
+                classname="inline checkbox-title"
+              />
+            </Checkbox>
+            {gaugeModuleChecked ? (
+              <div>
```

Comment:
I would prefer this div to be in a component (and same for token). `DumbModulesSection` job is to display the 2 checkboxes and to display the forms for the selected modules types